src/lib/locales/en.json

```
@@ -87,6 +87,22 @@
 	"upfront": "Upfront",
 	"percentage": "Percentage",
 	"fixedFee": "Fixed fee",
+	"login": "Login",
+	"forgotPassword": "Forgot password?",
+	"noAccount": "No account yet?",
+	"signUp": "Sign up",
+	"createAccount": "Create account",
+	"haveAccount": "Already have an account?",
+	"resetLink": "Send me a reset link",
+	"goBack": "Go back to ",
+	"updatePassword": "Update password",
+	"forgotPasswordText": "Enter the email associated with your account and you will receive a link to reset your password.",
+	"resetPasswordLocal": "If there’s an account associated {email} you will receive an email to your <a href=\"http://127.0.0.1:64324/m/{email}\" target=\"_blank\">inbucket</a> with instructions to reset your password.",
```

This does not look correct. It would be better to use a variable (`baseAddress` and define it somewhere):

```suggestion
	"resetPasswordLocal": "If there’s an account associated {email} you will receive an email to your <a href=\"{baseAddress}/m/{email}\" target=\"_blank\">inbucket</a> with instructions to reset your password.",
```